### ISSUE TITLE
fix minimumNVDAVersion

### ISSUE THREAD

**Issue by nishimotz** [nvdajp/focusHighlight#11]
FH does not work with the latest NVDA master branch.

```
ERROR - addonHandler._getAvailableAddonsFromPath (09:18:30.194):
Error loading Addon from path: C:\Users\nishimotz\AppData\Roaming\nvda\addons\focusHighlight
Traceback (most recent call last):
  File "addonHandler\__init__.pyo", line 236, in _getAvailableAddonsFromPath
  File "addonHandler\addonVersionCheck.pyo", line 173, in isAddonConsideredIncompatible
  File "addonHandler\addonVersionCheck.pyo", line 176, in isAddonConsideredCompatible
  File "addonHandler\addonVersionCheck.pyo", line 135, in getAddonCompatibility
  File "addonHandler\addonVersionCheck.pyo", line 119, in _getAutoDeduducedAddonCompat
  File "addonHandler\addonVersionCheck.pyo", line 158, in hasAddonGotRequiredSupport
  File "addonHandler\__init__.pyo", line 305, in minimumNVDAVersion
  File "versionInfo.pyo", line 46, in getNVDAVersionTupleFromString
ValueError: None
```
